README.md

```diff
@@ -62,39 +62,30 @@ There are three ways to do this:
 
 ## Examples
 
-The `examples` directory contains example measurements for MeasurementLink 2024 Q2 or later. If
-you are using a previous version of MeasurementLink, download the appropriate examples:
-
-- MeasurementLink 2023 Q1: [measurementlink-python-examples-1.0.1.zip](https://github.com/ni/measurementlink-python/releases/download/1.0.1/measurementlink-python-examples-1.0.1.zip)
```

Comment:
@dixonjoel I still think we should point users at the zip file and/or Git tag. I'll post a PR to restore this content.